src/ray/raylet/local_object_manager.cc

```diff
@@ -169,12 +169,10 @@ void LocalObjectManager::SpillObjectsInternal(
     }
     return;
   }
-
   io_worker_pool_.PopSpillWorker(
       [this, objects_to_spill, callback](std::shared_ptr<WorkerInterface> io_worker) {
         rpc::SpillObjectsRequest request;
         for (const auto &object_id : objects_to_spill) {
-          RAY_LOG(DEBUG) << "Sending spill request for object " << object_id;
```

Comment:
Can you add this back in? Actually I found this useful when debugging :)

Reply:
Sure! I don't know why I deleted it actually... haha